### PR TITLE
fix: Fixed the maximum desired number of sessions value in the Model Launcher modal to follow `max_session_count_per_model_session`

### PR DESCRIPTION
**Pull request by ironAiken2**
### This PR resolves [#2723 Issue](https://github.com/lablup/backend.ai-webui/issues/2723)

[teams](https://teams.microsoft.com/l/message/19:3c0acc0825024daeb2211f55c4e7f4aa@thread.skype/1727418901786?tenantId=13c6a44d-9b52-4b9e-aa34-0513ee7131f2&groupId=74ae2c4d-ec4d-4fdf-b2c2-f5041d1e8631&parentMessageId=1727418901786&teamName=devops&channelName=Backend.AI%20Talks&createdTime=1727418901786)

This PR introduces a new query to fetch the user's resource policy and applies the `max_session_count_per_model_session` limit to the service launcher page. The changes include:

- Added two new queries: `ServiceLauncherPageContent_UserInfoQuery` and `ServiceLauncherPageContent_UserResourcePolicyQuery`
- Utilized `useLazyLoadQuery` to fetch user information and resource policy
- Applied the `max_session_count_per_model_session` from the user's resource policy to the session count input field

These changes allow for dynamic session count limits based on the user's resource policy, improving the flexibility and customization of the service launcher.

**Checklist:**

- [ ] Documentation update for new resource policy feature
- [ ] Minimum required manager version: 23.09.6
- [ ] Test cases to verify the application of custom max session count limits

### CODE REVIEW

File: react/src/components/ServiceLauncherPageContent.tsx [hunk 1]
```diff
@@ -837,14 +874,19 @@ const ServiceLauncherPageContent: React.FC<ServiceLauncherPageContentProps> = ({
                       {
                         required: true,
                         min: 0,
-                        max: 10,
+                        max:
+                          user_resource_policy?.max_session_count_per_model_session ||
+                          10,
```
Comment (yomybaby) [inline react/src/components/ServiceLauncherPageContent.tsx]: When `user_resource_policy?.max_session_count_per_model_session` is `0`, the maximum will be set to 10 in the current code. Is this right?

**Review by yomybaby — CHANGES_REQUESTED** [review]

File: react/src/components/ServiceLauncherPageContent.tsx (hunk 1 above)
Comment (yomybaby) [reply react/src/components/ServiceLauncherPageContent.tsx]: If the value is 0, please use 0 as the max value in the UI. ([ref](https://teams.microsoft.com/l/message/19:3c0acc0825024daeb2211f55c4e7f4aa@thread.skype/1727686340501?tenantId=13c6a44d-9b52-4b9e-aa34-0513ee7131f2&groupId=74ae2c4d-ec4d-4fdf-b2c2-f5041d1e8631&parentMessageId=1727418901786&teamName=devops&channelName=Backend.AI%20Talks&createdTime=1727686340501))

Comment (ironAiken2) [reply react/src/components/ServiceLauncherPageContent.tsx]: Yep. For error handling, I'll set it to follow the default value of 0 if the value is undefined or null.

**Comment by ironAiken2** [comment]
When max value is 0, the error message was showing "value between 0 and 0", so I fixed that as well. @yomybaby

File: react/src/components/ServiceLauncherPageContent.tsx [hunk 2]
```diff
@@ -837,14 +874,25 @@ const ServiceLauncherPageContent: React.FC<ServiceLauncherPageContentProps> = ({
...
+                          user_resource_policy?.max_session_count_per_model_session ??
+                          0,
                         type: 'number',
+                        message:
+                          !user_resource_policy?.max_session_count_per_model_session ||
+                          user_resource_policy?.max_session_count_per_model_session ===
+                            0
+                            ? t('modelService.MaxRoutingCountIsZero')
+                            : undefined,
                       },
```
Comment (yomybaby) [inline react/src/components/ServiceLauncherPageContent.tsx]: Please separate the rules. 
```
rules={[
  { requilred:true },
  { type: number, min: 0},
  ....
]}
```

**Review by yomybaby — CHANGES_REQUESTED** [review]

**Review by yomybaby — APPROVED** [review]
LGTM, please check my commits.

**Pull request merged**